protocol/streaming/full_node_streaming_manager.go

```diff
@@ -424,9 +523,47 @@ func (sm *FullNodeStreamingManagerImpl) SendTakerOrderStatus(
 	)
 }
 
-// AddUpdatesToCache adds a series of updates to the full node streaming cache.
+// SendSubaccountUpdates groups subaccount updates by their subaccount ids and
+// sends messages to the subscribers.
+func (sm *FullNodeStreamingManagerImpl) SendSubaccountUpdates(
```

Comment:
We could just do `SendSubaccountUpdate` and not take in a series of subaccount updates. It doesn't quite make sense to have a couple subaccount updates in a row for the same subaccount id since we would only take the last one.

Reply:
Going to add this as a TODO for now. The current code replicates what's done by Indexer & the client code should cover the aggregation. Created https://linear.app/dydx/issue/CT-1117/aggregate-subaccount-update-per-subaccount-id to track